homeassistant/components/fujitsu_hvac/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""The Fujitsu HVAC (based on Ayla IOT) integration."""
+
+from __future__ import annotations
+
+from contextlib import suppress
+from dataclasses import dataclass
+
+from ayla_iot_unofficial import AylaAuthError, new_ayla_api
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+
+from .const import API_TIMEOUT, CONF_EUROPE, FGLAIR_APP_ID, FGLAIR_APP_SECRET
+from .coordinator import FujitsuHVACCoordinator
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+type FujitsuHVACConfigEntry = ConfigEntry[FujitsuHVACConfigData]
+
+
+@dataclass
+class FujitsuHVACConfigData:
+    """Config data for FujitsuHVAC config entries."""
+
+    coordinator: FujitsuHVACCoordinator
```

Comment:
```suggestion
type FujitsuHVACConfigEntry = ConfigEntry[FujitsuHVACCoordinator]
```
If you're only storing the coordinator, you don't have to wrap it